Derek branch

dcp14 opened the pull request.

dcp14 committed:
Specifying Remote

dcp14 committed:
Function Writing

Merge branch 'main' of https://github.com/UCLA-PHP/FluView

# Conflicts:
#	DESCRIPTION

dcp14 committed:
Date Update

dcp14 committed:
Removed Redundant Reactive

dcp14 committed:
Variant Fixing

d-morrison commented:
@dcp14 - try the "resolve conflicts" button above? We can go over the details together on Thursday.

dcp14 committed:
CDC Error Fixing

dcp14 committed:
Merge branch 'main' into derekBranch

d-morrison committed:
some tidying

d-morrison committed:
more

Merge branch 'main' into mergeDBranch

# Conflicts:
#	R/app_server.R

d-morrison committed:
more work

d-morrison committed:
fixed issue with date ordering

d-morrison committed:
fixed bug with NAs in simplified handling of strain selection

d-morrison committed:
removed block on loading from CDC

d-morrison committed:
cleaner style

d-morrison committed:
removed demo code

d-morrison committed:
created a package documentation page using use_package_doc()

d-morrison committed:
updated backup data

dcp14 committed:
Merge pull request #2 from UCLA-PHP/mergeDBranch

Merge Ezra's edits into derekBranch

dcp14 committed:
Before mergeDBranch Merge

dcp14 committed:
Has unresolved Conflicts

Merge branch 'derekBranch' of https://github.com/UCLA-PHP/FluView into derekBranch

# Conflicts:
#	R/app_server.R
#	R/app_ui.R

dcp14 committed:
has unresolved conflicts

d-morrison commented on R/app_server.R:
```diff
@@ -40,37 +35,141 @@ app_server <- function(input, output, session) {
+      temp
+    } |> reactive()
+
+  #message("Ready to Analyze", Sys.time())
```
Clean out the commented-out code?

d-morrison commented on data-raw/testing-tryCatch.R:
let's remove this file?

d-morrison reviewed (changes requested):
I finally circled back to this PR; looks good over all! Please make the changes that I requested in comments and then re-request review.

dcp14 committed:
Testing (NEEDS TO FIGURE OUT DIFFERENCE BETWEEN OLD AND NEW CODE)

dcp14 committed:
Back to What it is on Github (derekBranch)

dcp14 committed:
deleted testing-tryCatch
